### PR TITLE
Fix error when HostConfig is not defined in API request

### PR DESCRIPTION
**Pull request by georgyturevich**
Fix error when HostConfig is not defined in API request (e.g `sudo curl -H "Content-Type: application/json" -X POST -d '{"Image":"ubuntu"}' http://localhost:2375/containers/create` )

### CODE REVIEW

File: docker_enforcer.py [hunk 1]
```diff
@@ -252,6 +252,7 @@ def make_container_periodic_check_compatible(cont_json, url, owner):
     cont_json["Name"] = "<unnamed_container>" if "name" not in url_params else url_params["name"][0]
     cont_json["Config"] = {}
     cont_json["Config"]["Labels"] = cont_json.get("Labels", [])
+    cont_json["HostConfig"] = {} if "HostConfig" not in cont_json else cont_json["HostConfig"]
```
Comment (piontec) [inline docker_enforcer.py]: does it really make sense to add empty `HostConfig` here? maybe we should just refuse and return error? I think that docker CLI always puts some reasonable defaults there, even if default parameters are used. Other API clients should do as well, right? I don't remember now from where do we have this example of non-existing `HostConfig`...?

**Review by piontec — CHANGES_REQUESTED** [review]
please rebase against current `develop`

**Comment by georgyturevich** [comment]
@piontec Lukasz, I have rebased the PR just now. Could you merge it?

File: docker_enforcer.py [hunk 2]
```diff
@@ -252,9 +252,10 @@ def log_authz_req(json_data):
 def make_container_periodic_check_compatible(cont_json, url, owner):
     url_params = parse.parse_qs(url.query)
     cont_json["name"] = "<unnamed_container>" if "name" not in url_params else url_params["name"][0]
+    cont_json = docker_helper.rename_keys_to_lower(cont_json)
     cont_json["config"] = {}
     cont_json["config"]["labels"] = cont_json.get("labels", [])
-    cont_json = docker_helper.rename_keys_to_lower(cont_json)
+    cont_json["hostconfig"] = cont_json.get("hostconfig", {})
```
Comment (georgyturevich) [inline docker_enforcer.py:258]: @piontec Hi Lukasz,

I rebased the branch again. During rebasing I found one possible bug. We had `cont_json.get("labels", [])` before lowering letters. So most likely cont_json would contain `Labels`. So I moved `docker_helper.rename_keys_to_lower(cont_json)` two lines upper.

I also simplified cont_json["hostconfig"] initialization. I replaced `{} if "hostconfig" not in cont_json else cont_json["hostconfig"]` with `cont_json.get("hostconfig", {})`

**Pull request merged**